src/com/clover/remote/client/CloverConnector.ts

```diff
@@ -1526,26 +1562,28 @@ export namespace CloverConnector {
             this.cloverConnector.broadcaster.notifyOnRetrievePaymentResponse(response);
         }
 
-        public onRetrievePrintersResponse(result: sdk.remotepay.ResponseCode, reason: string, printers: sdk.printers.Printers[]): void {
+        public onRetrievePrintersResponse(result: sdk.remotepay.ResponseCode, printers: Array<sdk.printer.Printer>): void {
```

Comment:
Then in the future it can be added.  At this point it is a bit misleading as it will never be populated because the response object doesn't have a "reason".

Reply:
👍